kuma/settings/common.py

```diff
@@ -1211,6 +1217,12 @@ def pipeline_one_scss(slug, **kwargs):
 _PROD_ATTACHMENT_ORIGIN = 'demos-origin.mdn.mozit.cloud'
 ATTACHMENT_ORIGIN = config('ATTACHMENT_ORIGIN', default=_PROD_ATTACHMENT_ORIGIN)
 
+BETA_HOST = config('BETA_HOST', default='beta.' + DOMAIN)
+BETA_ORIGIN = config('BETA_ORIGIN', default='beta.mdn.mozit.cloud')
+BETA_SITE_URL = PROTOCOL + BETA_HOST
+WIKI_HOST = config('WIKI_HOST', default='wiki.' + DOMAIN)
```

Comment:
I think it's a subtle back-end thing (so I don't expect it to be obvious), but the `BETA_ORIGIN` is the domain used by the CloudFront CDN to point to the backing service (the ELB fronting the Django-based `web` pods) but using a different domain name so it can still be distinguished from standard requests even if we bypass the CDN (e.g., for basic testing). I don't think we're going to use a CDN for the `wiki` domain, so I don't see us needing a `WIKI_ORIGIN` setting that's different than the `WIKI_HOST` setting.